performanceplatform/collector/ga/contrib/content/table.py

```diff
@@ -52,3 +44,19 @@ def gen():
         documents.extend(ds)
 
     Pusher(data_set_config, options).push(documents)
+
+
+def get_department(filters):
+    for f in filters:
+        try:
+            filter_key, filter_value = re.split('=~\^', f)
+            return try_get_department(filter_value)
+        except:
+            raise ValueError("department not found in filters, expected "
+                             "filter expression expression containing "
+                             "'=~^' e.g. 'Organisation=~^<A1>'.")
+    else:
+        # If no matches, raise a RuntimeError
+        raise RuntimeError("department not found in filters, expected "
```

Comment:
@jcbashdown Which do you mean? The Runtime error has been left as it was. I added the Value error because I'm trying to split the filter into two variables.